### PR TITLE
v9 Migration product_brand

### PR DESCRIPTION
**Pull request by JayVora-SerpentCS**

### CODE REVIEW

File: product_brand/__openerp__.py [hunk 1]
```diff
@@ -30,16 +33,18 @@
...
-    'version': '8.0.0.1.0',
+    'version': '9.0.0.1.0',
     'category': 'Product',
     'summary': 'Add brand to products',
     'author': 'NetAndCo, Akretion, Prisnet Telecommunications SA'
-              ', MONK Software, Odoo Community Association (OCA)',
-    'license': 'AGPL-3',
+              ', MONK Software, Odoo Community Association (OCA)'
+              ', Serpent Consulting Services Pvt. Ltd.',
+    'license': 'LGPL-3',
```
Comment (nhomar) [inline product_brand/__openerp__.py]: Why change to LGPL?

**Comment by nhomar** [comment]
:+1:  for me just do not change the licence pliz, this simple module is fine AGPL and it was not asked to change.

Regards.

**Comment by JayVora-SerpentCS** [comment]
License change was insane, should be AGPL always. Corrected myself.

**Comment by elicoidal** [comment]
@JayVora-SerpentCS if you are gonna migrate the module to v9, please adapt README and headers to the new standards:
- Please follow the OCA README template:
  https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
- We recommend to include a screenshot as well to help new comers (nice to have)
- Please use simple headers in all py files following OCA templates
  https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

File: product_brand/README.rst [hunk 2]
```diff
@@ -10,7 +10,7 @@ Installation
 
 To install this module, you need to:
 
- * git clone https://github.com/OCA/product_attribute --branch 8.0
+ * git clone https://github.com/OCA/product_attribute --branch 9.0
```
Comment (elicoidal) [inline product_brand/README.rst]: this is not necessary and very specific. Please remove these lines.

File: product_brand/README.rst [hunk 3]
```diff
@@ -10,7 +10,7 @@ Installation
```
Comment (elicoidal) [inline product_brand/README.rst]: remove "[product_brand]" from title

File: product_brand/README.rst [hunk 4]
```diff
@@ -59,6 +59,7 @@ Contributors
 * Seraphine Lantible <s.lantible@gmail.com>
 * Gunnar Wagner <vrms@netcologne.de>
 * Leonardo Donelli <donelli@webmonks.it>
+* Serpent Consulting Services Pvt. Ltd. <jay.vora@serpentcs.com>
 
 Maintainer
 ----------
```
Comment (elicoidal) [inline product_brand/README.rst:71]: below: https (x2)

File: product_brand/__init__.py [hunk 5]
```diff
@@ -5,10 +5,13 @@
 # Copyright (C) 2009 NetAndCo (<http://www.netandco.net>). #                  #
 # Copyright (C) 2011 Akretion Benoît Guillot <benoit.guillot@akretion.com> #  #
 # Copyright (C) 2014 prisnet.ch Seraphine Lantible <s.lantible@gmail.com> #   #
-# Contributors                                                                #
```
Comment (elicoidal) [inline product_brand/__init__.py:8]: short headers: usually no contributors (they are in README)

File: product_brand/__openerp__.py [hunk 6]
```diff
@@ -30,16 +33,18 @@
 ###############################################################################
 {
     'name': 'Product Brand Manager',
-    'version': '8.0.0.1.0',
+    'version': '9.0.0.1.0',
```
Comment (elicoidal) [inline product_brand/__openerp__.py]: 9.0.1.0.0

File: product_brand/models/__init__.py [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+# This program is distributed in the hope that it will be useful, #           #
+# but WITHOUT ANY WARRANTY; without even the implied warranty of #            #
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the #              #
+# GNU Affero General Public License for more details. #                       #
+# #                                                                           #
+# You should have received a copy of the GNU Affero General Public License #  #
+# along with this program. If not, see <http://www.gnu.org/licenses/>. #      #
+# #                                                                           #
+###############################################################################
+###############################################################################
```
Comment (elicoidal) [inline product_brand/models/__init__.py]: remove

**Comment by JayVora-SerpentCS** [comment]
Noted, work in progress..

File: product_brand/README.rst (hunk 4 above)
Comment (JayVora-SerpentCS) [inline product_brand/README.rst:71]: @elicoidal Sorry, didnot get that.

**Comment by JayVora-SerpentCS** [comment]
We will do product_qt too. In fact, have done it. Should we add here or make another pr?

**Comment by pedrobaeza** [comment]
In another PR, please. Which module is that? For annotating on the issue that tracks the progress: https://github.com/OCA/product-attribute/issues/100

**Comment by JayVora-SerpentCS** [comment]
product_qt module.

**Comment by pedrobaeza** [comment]
It definitely needs a better name. Anyway, did you see the set of quality_control modules in OCA/manufacture? I think they are far better than this one.

**Comment by JayVora-SerpentCS** [comment]
I guess Quality_control is much better as I have used in one MRP setup.

**Comment by JayVora-SerpentCS** [comment]
Wondering why Runbot fails, any hints?

**Comment by ghost** [comment]
@pedrobaeza @elicoidal Can you please help to resolve travis and runbot fails ?

Regards.

**Comment by pedrobaeza** [comment]
The problem is due to a long author name (more than 128), which avoids the installation of the module. Can you shorten a bit your name?

This is the issue that I opened some time ago about the topic: https://github.com/odoo/odoo/issues/5850

**Comment by ghost** [comment]
@pedrobaeza Thanks for the prompt response.

I have shorten the name and it worked +1.
We are also planning to propose new PR for  website_sale_product_brand in v9 which is dependent on this product_brand module.

**CCing** @JayVora-SerpentCS @nhomar @elicoidal Can you please review the changes?

Thanks.

**Comment by pedrobaeza** [comment]
@MeetSCS, OK, I have updated the corresponding issue: https://github.com/OCA/e-commerce/issues/63

**Comment by ghost** [comment]
@pedrobaeza Thanks for the update.

Waiting for this module to merge.

Regards.

**Comment by ghost** [comment]
@pedrobaeza @elicoidal @nhomar 

Any updates on this PR ?

When can we merge this PR ?

Regards.

File: product_brand/README.rst [hunk 8]
```diff
@@ -1,32 +1,34 @@
-Product Brand Manager [product_brand]
-=====================================
+.. image:: https://img.shields.io/badge/license-AGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl.html
+   :alt: License: AGPL-3
 
-This module allows odoo users to easily manage product brands. You can
-define brands, attach a logo and a description to them. 
-It also allows to attach a partner to a brand.
+Product Brand Manager
```
Comment (elicoidal) [inline product_brand/README.rst:6]: add a line before

File: product_brand/README.rst [hunk 9]
```diff
@@ -1,32 +1,34 @@
...
+======================
```
Comment (elicoidal) [inline product_brand/README.rst]: make sure the line is exactly the same length of the title

File: product_brand/README.rst [hunk 10]
```diff
@@ -1,32 +1,34 @@
...
+
+ No configuration is required.
 
 Usage
 =====
 
 To use this module, you need to:
 
- * go to ...
+ * Go to ...
```
Comment (elicoidal) [inline product_brand/README.rst]: remove this empty section

File: product_brand/README.rst [hunk 11]
```diff
@@ -1,32 +1,34 @@
...
 
 For further information, please visit:
```
Comment (elicoidal) [inline product_brand/README.rst:31]: remove this part

File: product_brand/README.rst (hunk 4 above)
Comment (elicoidal) [inline product_brand/README.rst:71]: below url should be https instead of http

File: product_brand/models/product_brand.py [hunk 12]
```diff
@@ -0,0 +1,47 @@
...
+
+class ProductBrand(models.Model):
+    _name = 'product.brand'
+
+    name = fields.Char('Brand Name', required=True)
+    description = fields.Text('Description', translate=True)
+    partner_id = fields.Many2one(
+        'res.partner',
+        string='Partner',
+        help='Select a partner for this brand if it exists',
```
Comment (elicoidal) [inline product_brand/models/product_brand.py]: s/if it exists/if any

**Comment by elicoidal** [comment]
mostly details on my side. :+1:

**Comment by ghost** [comment]
@elicoidal  Thanks for the review, i will update the changes soon.

**Comment by ghost** [comment]
CCing @elicoidal @pedrobaeza @nhomar @JayVora-SerpentCS Travis and other checks passed :+1:

File: product_brand/README.rst (hunk 8 above)
Comment (pedrobaeza) [inline product_brand/README.rst:6]: It must be a line with `====`

**Comment by alan196** [comment]
:+1:

**Comment by ghost** [comment]
Any updates on this PR ?

**Comment by pedrobaeza** [comment]
You only have to add the line `======...` above the title and I'll merge.

**Comment by ghost** [comment]
@pedrobaeza Is it ok now ?

Regards.

**Comment by ghost** [comment]
:+1:

**Pull request merged**